app/main/index.ts

```diff
@@ -1,21 +1,25 @@
 'use strict';
-const path = require('path');
-const fs = require('fs');
-
-const electron = require('electron');
-const windowStateKeeper = require('electron-window-state');
-const isDev = require('electron-is-dev');
-const appMenu = require('./menu');
-const { appUpdater } = require('./autoupdater');
+import { sentryInit } from '../renderer/js/utils/sentry-util';
+import { appUpdater } from './autoupdater';
+import { setAutoLaunch } from './startup';
+
+import windowStateKeeper = require('electron-window-state');
+import path = require('path');
```

Comment:
Is there a specific reason for using `require` or `from` (or the other way around?)
It would make things much simpler if we decided on using only one.

Reply:
@kanishk98 yes! So the problem is we can use the from syntax to import modules that either export a class or function (eventhough for node both are functions!). So instead of having a mix we decided to use the require syntax for node modules and for our own code we still use from.

Reply:
Ah right, makes sense. Thanks!